collections.go

```diff
@@ -17,6 +17,11 @@ func newCollectionsClient(sl *sling.Sling) *CollectionsClient {
 	return &CollectionsClient{sl: sl}
 }
 
+// Structure gives access to id's document structure.
+func (cl *CollectionsClient) Structure(id CollectionID) *CollectionsStructureClient {
```

Comment:
What about renaming it to
```suggestion
func (cl *CollectionsClient) DocumentStructure(id CollectionID) *CollectionsDocumentStructureClient {
```
? And all the other related functions and structs as well? 🤔

Reply:
sure :+1: